api/daemon.go

```diff
@@ -57,6 +54,39 @@ bwIDAQAB
 // manifest.
 var updateURL = "http://23.239.14.98/releases/" + runtime.GOOS + "_" + runtime.GOARCH
 
+// SiaConstants is a struct listing all of the constants in use.
+type SiaConstants struct {
```

Comment:
well, it does have the advantage of making it clear that MaturityDelay is a constant and not a type...
I'll drop it for now